### PR TITLE
Firebase talk description update

### PR DESCRIPTION
**Pull request by HHogg**

### CODE REVIEW

File: _posts/2018-01-04-firebase.md [hunk 1]
```diff
@@ -26,4 +26,20 @@ layout: event.html
 collection: events
 ---
 
-Harry Hogg returns to Async to give us the low down on [Firebase](https://firebase.google.com/), Google's insta-backend. More details to follow.
+We've all had that night where we get a great idea for a side project, and to make it really great it needs to have authentication, persistent state, some sort of backend to handle the stuff we can't expose to the frontend. If you think it's a really great idea you might even throw is some analytics.
```
Comment (larister) [inline _posts/2018-01-04-firebase.md]: `throw *in* some analytics` :)

File: _posts/2018-01-04-firebase.md [hunk 2]
```diff
@@ -26,4 +26,20 @@ layout: event.html
...
+
+You start off by looking at what's the ~cheapest~ best server you can spin up... is it a Droplet or a Dyno this month? Ubuntu or CentOS? Probably need some Docker, Chef or Puppet to make sure I always redeploy it correctly. Now I can make a start on the Rest API... ah yeah Express, KOA? MongoDB connector, is Mongoose still a thing? Don't forget to set up those JWTs correctly.
+
+At this point your still having fun... who doesn't like building a clean, secure and well documented REST API and hitting it up with Postman (those automated testing features are amazing).
```
Comment (larister) [inline _posts/2018-01-04-firebase.md]: `you're still having fun...` ;)

File: _posts/2018-01-04-firebase.md [hunk 3]
```diff
@@ -26,4 +26,20 @@ layout: event.html
...
+
+Seriously it's really rewarding to get all that set up and working, but by the time I get to the building the idea I'm all funned out and it just becomes another directory in my projects folder.
+
+Google's Firebase suite of tools get you up and running with *everything* in minutes, so you can focus on that awesome life changing idea.
+
+Come down for a run down on the tools to get your web app up and running.
+
+Bring a laptop if you can, we'll be forming into teams to put your Firebase skills to the test in a chain reaction game to be the first team to send your rocket to Mars.
```
Comment (larister) [inline _posts/2018-01-04-firebase.md]: Do you mind putting `Bring a laptop` in `**bold**`?

**Comment by HHogg** [comment]
Thanks @larister all updated!

**Pull request merged**